src/main/scala/cromwell/engine/store/SymbolStore.scala

```diff
@@ -0,0 +1,86 @@
+package cromwell.engine.store
+
+import cromwell.binding._
+import cromwell.binding.types.WdlType
+import cromwell.binding.values.{WdlObject, WdlValue}
+import cromwell.engine.store.SymbolStore._
+
+import scala.util.{Failure, Success, Try}
+
+object SymbolStore {
+  case class SymbolStoreKey(scope: String, name: String, iteration: Option[Int], input: Boolean)
+
+  object CallInputWdlFunctions extends WdlFunctions {
+    def getFunction(name: String): WdlFunction = {
```

Comment:
Discussed in person. this is incorrect but we think it's an implementation of something that's perhaps more fundamentally incorrect - it's out of scope for any of these stories and leaving it as-is for now.